testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/api/SocketsAndInterfacesTestCase.java

```diff
@@ -129,9 +154,7 @@ public void testAddUpdateRemove() throws Exception {
 
         logger.info("Restarting server.");
 
-        // reload server
-        op = createOpNode(null, "reload");
-        result = executeOperation(op);
+        ServerReload.executeReloadAndWaitForCompletion(getModelControllerClient());
```

Comment:
all resources you are testing are runtime addable/removable so reload is not needed at all.